addons/apecloud-mysql/scripts/etcd-post-start.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
-endpoints=${ETCD_SERVER:-'127.0.0.1:2379'}
+if [ -n "$SERVICE_ETCD_ENDPOINT" ]; then
+  endpoints="$SERVICE_ETCD_ENDPOINT"
+else
+  echo "SERVICE_ETCD_ENDPOINT is empty. Cannot proceed."
+  exit 1
+fi
+
 cell=${CELL:-'zone1'}
```

Comment:
if multiple Clusters share the same etcd, $CELL will conflict.

Reply:
if you want wescale shares etcd, its zone should be a value that contains cluster name.

Reply:
vtctl --topo_implementation etcd2 \
  --topo_global_server_address "${servers}" \
  --topo_global_root "/vitess/${KB_NAMESPACE}/${KB_CLUSTER_NAME}/global" VtctldCommand AddCellInfo \
  --root "/vitess/${KB_NAMESPACE}/${KB_CLUSTER_NAME}/$cell" \
  --server-address "${servers}" \
  $cell
set -e

As with the parameter settings of vtctl, all etcd-related configurations include the cluster name and namespace.

Reply:
I see